### ISSUE TITLE
Custom.py fails

### ISSUE THREAD

**Issue by code3z** [code3z/vivaldi_mods_collection#2]
I will be asking lonm about this and will figure it out.

**Comment by code3z** [comment]
`home@chrx:~/vivaldi_mods_collection$ sudo python3 custom.py -i config.json
WARNING - Uninstalling old mods
WARNING - Installing mods
ERROR - Failed to copy mod lonm/VivaldiMods/mods/advancedPanels.js
ERROR - Failed to copy mod lonm/VivaldiMods/mods/autoSaveSessions.js
ERROR - Failed to copy mod luetage/vivaldi_modding/import-export_themes.js
ERROR - Failed to copy mod luetage/vivaldi_modding/backup-search-engines.js
ERROR - Failed to copy mod lonm/VivaldiMods/mods/panel_actions.js
`

**Comment by ortiza5** [comment]
That error makes me think that those files don't actually exist. Those should be file paths relative to the directory that contains `custom.py` and `config.json`.

**Comment by code3z** [comment]
They do exist, but unfortunately did not get committed to the repository. I am working on that. If you look at the error it is only the JS mods that throw errors. The script handles them differently. 


Also I will add your modding repository to this, if that's okay. Does it have a license on it?




That error makes me think that those files don't actually exist. Those should be file paths relative to the directory that contains custom.py and config.json.

**Comment by ortiza5** [comment]
Are those files in a folder called `mods` in the same directory that contains `custom.py` and `config.json`? That is where the script is expecting to find them.

> Also I will add your modding repository to this, if that's okay. Does it have a license on it?

You can add any of my mods, but I am still updating some info in my repository. There is no license.

**Comment by ortiza5** [comment]
It also might be a file permissions issue?

**Comment by code3z** [comment]
No, they are in subdirectories. I thought it handled subdirectories but I could be wrong.
No liscense means a strict copyright in most countries. If you don't care how people use your mods you just need a "no rights reserved" and a disclaimer, see unliscense.org.



Are those files in a folder called mods in the same directory that contains custom.py and config.json? That is where the script is expecting to find them.

Also I will add your modding repository to this, if that's okay. Does it have a license on it?

You can add any of my mods, but I am still updating some info in my repository. There is no license.

**Comment by code3z** [comment]
I think it may be an issue with incorrectly embedding another git repository.
